DragAndDrop/DraggableView.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Drag.swift
+//  DragAndDrop
+//
+//  Created by Stanley Chiang on 3/6/16.
+//  Copyright © 2016 SantaClaraiOSConnect. All rights reserved.
+//
+
+import UIKit
+
+protocol draggableViewDelegate {
+    func didDragToTrash() -> Bool
+}
+
+class DraggableView: UIView {
+    
+    var panGestureRecognizer:UIPanGestureRecognizer!
```

Comment:
Would love to see this not be an ImplicitlyUnwrappedOptional :smile:

For this one specifically we can just initialize it directly. 
`let panGestureRecognizer = UIPanGestureRecognizer()`
Then remove the declaration from `init()`